lib/openxml/drawingml/elements/paragraph.rb

```diff
@@ -0,0 +1,74 @@
+module OpenXml
+  module DrawingML
+    module Elements
+      class Paragraph < OpenXml::DrawingML::Element
+        include HasChildren
+        include HasProperties
+
+        tag :p
+
+        # Validations
+        COORDINATE_REGEX = /^(-?[0-9]+(:?\.[0-9]+)?)(:?mm|cm|in|pt|pc|pi)?$/
+
+        coordinate = ->(value) { raise ArgumentError unless value.to_s =~ coordinate_regex }
+        indent_range = lambda { |value|
+          matched = value.to_s =~ COORDINATE_REGEX
+          raise ArgumentError unless matched || (-51_206_400..51_206_400).cover?(value)
+        }
+        text_margin = lambda { |value|
+          matched = value.to_s =~ COORDINATE_REGEX
+          raise ArgumentError unless matched || (0..51_206_400).cover?(value)
```

Comment:
Should these `ArgumentError`s have a message?

Reply:
Yeah, probably so...